packages/contracts/scripts/deploy/dollar/solidityScripting/TWAPOracle.s.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import "./UbiquityGovernanceToken.s.sol";
+
+contract TWAPScript is GovernanceScript {
+	function run() public virtual override {
+		super.run();
+		uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
```

Comment:
I wish that we could dynamically parse the code and render a list of required environment variables in the `.env.example` file!